sigs.yaml

```diff
@@ -665,14 +665,22 @@ sigs:
       url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
       archive_url: https://docs.google.com/document/d/130_kiXjG7graFNSnIAgtMS1G8zPDwpkshgfRYS0nggo/edit
       recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP29D0nYgAGWt1ZFqS9Z7lw4
-    - description: Cluster API working group
+    - description: Cluster API office hours
       day: Wednesday
       time: "10:00"
       tz: "PT (Pacific Time)"
       frequency: weekly
       url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
       archive_url: https://docs.google.com/document/d/16ils69KImmE94RlmzjWDrkmFZysgB2J4lGnYMRN89WM/edit#
       recordings_url: https://www.youtube.com/playlist?list=PL69nYSiGNLP29D0nYgAGWt1ZFqS9Z7lw4
+    - description: Cluster API (AWS implementation) office hours
+      day: Monday
+      time: "10:00"
+      tz: "PT (Pacific Time)"
+      frequency: weekly
+      url: https://docs.google.com/document/d/10dq54Fd-xa6P5Iy3p46VY1YTFqugGMd1PygDIpuRw6c/edit
```

Comment:
@kris-nova ?  

I don't have strong opinions here.

Reply:
@detiber @kris-nova  - can someone highlander the notes and then I'll update here?

Reply:
@timothysc It looks like that doc is currently a placeholder and since the other meetings reference the same placeholder doc for their `url` I don't see an issue with just updating it.